docs: Improve NOTES.txt

viccuad opened the pull request:
## Description

<!-- Please provide the link to the GitHub issue you are addressing -->
Attempt to improve NOTES.txt that show on installation/upgrade.

While it isn't easy to summarize Kubewarden, we could do better here.

<!-- Please provide the link to the documentation related to your change, if applicable -->
<!-- [Documentation](https://<insert your url>) -->

## Test

<!-- Please provides a short description about how to test your pullrequest -->
<!--
```shell
cp <to_package_directory>
go test
```
-->

## Additional Information

### Tradeoff

<!-- Please describe, if any, the tradeoffs that you found acceptable in this pull request -->

### Potential improvement

<!-- Please describe, if any, potential improvement that you are envisioning -->

flavio commented on charts/kubewarden-defaults/templates/NOTES.txt:
```diff
@@ -1,7 +1,9 @@
 
-For more information on how to define policies, check out https://kubewarden.io/
+For more information check out https://docs.kubewarden.io/quick-start
+For a list of policies, check out https://artifacthub.io/packages/search?kind=13
```
```
Discover ready to use policies by visiting https://artifacthub.io/packages/search?kind=13
```

olblak commented:
I think we could leverage `if` statement in the NOTE.txt such as 

```
kubewarden-defaults installed.

You now have a PolicyServer running in your cluster ready to run any 
`clusteradmissionpolicies.policies.kubewarden.io` or
`admissionpolicies.policies.kubewarden.io` resources.

For more information on how to define policies, check out https://kubewarden.io/

{{ if .Values.recommendedPolicies }}

Installed recommended policies
* kubewarden/policies/allow-privilege-escalation-psp
-> https://artifacthub.io/packages/kubewarden/allow-privilege-escalation-psp/allow-privilege-escalation-psp
* kubewarden/policies/host-namespaces-psp
-> https://artifacthub.io/packages/kubewarden/host-namespaces-psp/host-namespaces-psp
* kubewarden/policies/pod-privileged
-> https://artifacthub.io/packages/kubewarden/pod-privileged-policy/pod-privileged-policy
* kubewarden/policies/user-group-psp
-> https://artifacthub.io/packages/kubewarden/user-group-psp/user-group-psp
* kubewarden/policies/hostpaths-psp
-> https://artifacthub.io/packages/kubewarden/hostpaths-psp/hostpaths-psp
* kubewarden/policies/capabilities-psp
-> https://artifacthub.io/packages/kubewarden/capabilities-psp/capabilities-psp

{{- end }}

```

viccuad commented:
Addressed feedback,  now it displays:

```console
$ helm upgrade -i --wait --namespace kubewarden --create-namespace kubewarden-controller ./charts/kubewarden-controller
Release "kubewarden-controller" has been upgraded. Happy Helming!
NAME: kubewarden-controller
LAST DEPLOYED: Tue Aug  1 12:15:11 2023
NAMESPACE: kubewarden
STATUS: deployed
REVISION: 4
TEST SUITE: None
NOTES:
You can now start defining admission policies by using the cluster-wide
`clusteradmissionpolicies.policies.kubewarden.io` or the namespaced
`admissionpolicies.policies.kubewarden.io` resources.

For more information check out https://docs.kubewarden.io.
$ helm upgrade -i --wait --namespace kubewarden --create-namespace kubewarden-defaults ./charts/kubewarden-defaults --set recommendedPolicies.enabled=True --set recommendedPolicies.defaultPolicyMode=monitor
Release "kubewarden-defaults" has been upgraded. Happy Helming!
NAME: kubewarden-defaults
LAST DEPLOYED: Tue Aug  1 12:14:19 2023
NAMESPACE: kubewarden
STATUS: deployed
REVISION: 9
TEST SUITE: None
NOTES:
You now have a `PolicyServer` named `default` running in your cluster.
It is ready to run any `clusteradmissionpolicies.policies.kubewarden.io` or
`admissionpolicies.policies.kubewarden.io` resources.

Installed recommended policies:
- no-privilege-escalation
  module: kubewarden/policies/allow-privilege-escalation-psp:v0.2.6
- no-host-namespace-sharing
  module: kubewarden/policies/host-namespaces-psp:v0.1.6
- no-privileged-pod
  kubewarden/policies/pod-privileged:v0.2.7
- do-not-run-as-root
  kubewarden/policies/user-group-psp:v0.4.9
- do-not-share-host-paths
  kubewarden/policies/hostpaths-psp:v0.1.9
- drop-capabilities
  kubewarden/policies/capabilities-psp:v0.1.13

For more information check out https://docs.kubewarden.io/quick-start.
Discover ready to use policies at https://artifacthub.io/packages/search?kind=13.
```

flavio reviewed (approved):
nice changes
